proto/api/v1/network_graph_service.proto

```diff
@@ -94,7 +104,14 @@ service NetworkGraphService {
   }
 
   rpc GetExternalNetworkFlows(GetExternalNetworkFlowsRequest) returns (GetExternalNetworkFlowsResponse) {
-    option (google.api.http) = {get: "/v1/networkgraph/cluster/{cluster_id}/externalentities/flows/{deployment_id}"};
+    option (google.api.http) = {get:
+        "/v1/networkgraph/cluster/{cluster_id}/externalentities/flows/"
+        "{deployment_id}"
+};
+  }
+
+  rpc GetFlowsByEntity(GetFlowsByEntityRequest) returns (GetFlowsByEntityResponse) {
+    option (google.api.http) = {get: "/v1/networkgraph/cluster/{cluster_id}/flows/{entity_id}"};
```

Comment:
I wonder if the path here leaves enough room to retrieve flows based on another criteria in the future.